docs/cloud/pouta/multiattach.md

```diff
@@ -61,163 +61,277 @@ The Global file system or (GFS2 in short) is a file system currently developed b
 
-$ for i in $(seq 1 16);
-do
-    openstack --os-compute-api-version 2.60 server add volume "cinder-gfs2-$i" multi-attach-test-gfs2
-done
+1. For AlmaLinux and other RedHat based distributions you just need enable two collections and install few packages **on every nodes:**
 
-$ ansible-playbook main.yml -e fs='gfs2'
-```
+    ```sh
+    $> dnf config-manager --enable  highavailability resilientstorage
+    $> dnf install pacemaker corosync pcs dlm gfs2-utils lvm2-lockd
+    ```
 
-You need to run Ansible twice due to a bug in the `openstack.cloud.server_volume` which can only attach the volume to a single VM and fails with the other ones.
+#### Cluster setup
+!!! warning "root user"
+    The following commands are executed as the root user  
+    It will be specified throughout this tutorial if the commands must be run on a single or every nodes.
 
-If you already have a cluster of VMs, or want to manually create them, it is still possible to use the `gfs2` Ansible role. The steps are simple:
+1. Run the following commands **on every nodes:**
 
-1. Create and attach the volume. See the manual [Create and attach a volume](#create-and-attach-a-volume) from above.
+    ```sh
+    $> systemctl start pcsd.service
+    $> systemctl enable pcsd.service
+    ```
 
-1. Create a standard Ansible inventory like this one:
+1. When you install `pacemaker`, it creates an user named `hacluster`. You need to set a password to this user:
+   
+    ```sh
+    $> passwd hacluster
+    ```
 
-    ```ini
-    [all]
-    <VM_name> ansible_host=192.168.1.XXX ansible_user=<user>
-    # ...
-    [all:vars]
-    ansible_ssh_common_args='-J <jumphost>'
+1. Make sure that every node **domain name** can be resolved in every other node. In Pouta, the simplest way is to use [/etc/hosts](https://en.wikipedia.org/wiki/Hosts_(file)), where each host has a line similar to:
+
+    ```sh
+    <ip> <vm_name>
+    ```
+
+1. Run the following commands **only on one node**:
+
+    ```
+    $> pcs host auth node1 node2 node3 ...
+    Username: hacluster
+    Password: *******
+    $> pcs cluster setup <cluster_name> node1 node2 node3 ...
+    $> pcs cluster start --all
+    ```
+
+1. You can check the status by running the commands:
+
+    ```sh
+    $> pcs cluster status
+    $> pcs status corosync
+    ```
+
+By default, `corosync` and `pacemaker` services are disabled:
+
+    $> pcs status
+    Daemon Status:
+      corosync: active/disabled
+      pacemaker: active/disabled
+      pcsd: active/enabled
+    
+According to [pacemaker docs](https://clusterlabs.org/pacemaker/doc/2.1/Clusters_from_Scratch/html/verification.html):
+
+    requiring a manual start of cluster services gives you the opportunity to do a post-mortem investigation of a node failure before returning it to the cluster.
+
+That means, if a node crash and restart, you have to start the two services and then run the command `pcs cluster start <node>`.  
+You can enable them if you wish.
```

Comment:
How?

Please add the example command. Users might be confused if `pcs` is needed to enable the service, or if `systemctl` is the one needed.

Reply:
Also, it is not clear for me if that by enabling the service, it will be started after a crash.

Reply:
It's mentioned in the link of the pacemaker doc:
```
Important

In this example, we are not enabling the corosync and pacemaker services to start at boot. If a cluster node fails or is rebooted, you will need to run pcs cluster start [<NODENAME> | --all] to start the cluster on it. While you can enable the services to start at boot (for example, using pcs cluster enable [<NODENAME> | --all]), requiring a manual start of cluster services gives you the opportunity to do a post-mortem investigation of a node failure before returning it to the cluster.
```

Reply:
I added only `pcs` command and made it clearer